tests/vk-regression/vk-regression.json

```diff
@@ -50,16 +50,16 @@
     }
   },
   "HelloWorld": {
-    "digest": "11c21044c0ab297f29b386d1d04d0318cff4effa75191371dd576282882c2a8d",
+    "digest": "103e8bd9af45de70a1b214377e71bd05ae464e932dbe48aeada70362fa38b05",
     "methods": {
       "update": {
-        "rows": 826,
-        "digest": "5215ab9c7474b26f3a9073d821aef59b"
+        "rows": 825,
+        "digest": "226ff56f432f39d8056bf7f1813669f5"
```

Comment:
1-row reduction makes sense because the hello-world example does `privateKey.toPublicKey()` which as the final step converts the `Group` (uncompressed EC point `(x,y)`) into a `PublicKey` (compressed EC point `(x, isOdd)`) and to do that it needs to get all of `y`'s bits with `toBits()`. `toBits()` needs 1 row less now because it adds up 254 bits, not 255. 

It does mean that certain public keys are no longer supported by this method. I think we should address that separately by introducing a dedicated `parity()` gadget using range checks, which makes it more efficient as well

Reply:
I had wondered about this. Would it make sense to have this use `toFullBits()` here in the meantime?

Reply:
Yes that would make sense